### PR TITLE
Fixed cat painting step 47 missing comma #55989

### PR DESCRIPTION
**Pull request by VDXD**
Checklist:

- [X ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X ] My pull request targets the `main` branch of freeCodeCamp.
- [ X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This is my first contribution i hope i will get chance to help more in future
Thank you

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646de7b64467e96b7d35b5cd.md [hunk 1]
```diff
@@ -154,9 +154,9 @@ body {
 }
 
 .cat-right-eye {
-  --fcc-editable-region--
-  
-  --fcc-editable-region--
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646de7b64467e96b7d35b5cd.md:159]: This is not the requested change, this is breaking the step. Revert this change.

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Thank you for your PR, but this is not what the issue ask for.
